lib/matplotlib/backends/__init__.py

```diff
@@ -60,3 +52,13 @@ def do_nothing(*args, **kwargs): pass
     matplotlib.verbose.report('backend %s version %s' % (backend,backend_version))
 
     return backend_mod, new_figure_manager, draw_if_interactive, show
+
+def do_nothing_show(*args, **kwargs):
```

Comment:
unless it is used else where, I think this should stay as a nested function.  No reason to leak it into the top level name space.